src/local-peers.js

```diff
@@ -44,6 +53,12 @@ class Peer {
   #connected
   /** @type {Map<string, Array<DeferredPromise<InviteResponse['decision']>>>} */
   pendingInvites = new Map()
+  /** @type {string | undefined} */
+  #name
+  #connectedAt = 0
+  #disconnectedAt = 0
```

Comment:
is it too inconvenient to initialize with null? minor concern is that using this technically means January 1, 1970, UTC and having to know that `0` is a magic number for us in this case.